clusterloader2/pkg/prometheus/manifests/prometheus-windows-scrape-configs.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
 stringData:
   windows-scrape-configs.yaml: |-
     - job_name: "monitoring/windows_static"
-      scrape_interval: {{DefaultParam .CL2_WINDOWS_SCRAPE_INTERVAL "130s"}}
-      scrape_timeout: {{DefaultParam .CL2_WINDOWS_SCRAPE_TIMEOUT "120s"}}
+      scrape_interval: {{DefaultParam .CL2_WINDOWS_SCRAPE_INTERVAL "1m"}}
+      scrape_timeout: {{DefaultParam .CL2_WINDOWS_SCRAPE_TIMEOUT "10s"}}
```

Comment:
Consider leaving this a bit higher, just in case? Maybe 30s or 45s would still be fine.

Reply:
10s is still not particularly low. I think we can try 10s and fix if it will still cause issues.

Reply:
ok